src/main/java/com/puppycrawl/tools/checkstyle/utils/TokenUtil.java

```diff
@@ -353,4 +353,18 @@ public static BitSet asBitSet(String... tokens) {
                 .collect(BitSet::new, BitSet::set, BitSet::or);
     }
 
+    /**
+     * Checks if the given identifier AST node is an unnamed variable.
+     * An unnamed variable is a variable that is just an underscore {@code _}.
+     * A local variable, exception parameter, or lambda parameter that is declared using
+     * an underscore is called an unnamed local variable, unnamed exception parameter,
+     * or unnamed lambda parameter, respectively.
+     *
+     * @param identifierAst the AST node representing the identifier
+     * @return true if the identifier is an unnamed variable, false otherwise
+     */
+    public static boolean isUnnamed(DetailAST identifierAst) {
+        return "_".equals(identifierAst.getText());
+    }
```

Comment:
Let's wait for utility until we need this in more than one place

Reply:
This condition is no simple, I am not even sure it is worthy of having a utility method

Reply:
ok, yes it is very simple I just put it there because we will need to use it a lot instead of putting it in the checks classes

Reply:
@nrmancuso done